UnityProject/Assets/Scripts/HealthV2/Living/Respiration/RespiratorySystemBase.cs

```diff
@@ -26,6 +28,8 @@ public class RespiratorySystemBase : MonoBehaviour //Not really a Respiratory Mo
 
 		public bool IsSuffocating => healthStateController.IsSuffocating;
 
+		public readonly SyncList<BreathingTubeImplant> CurrentBreathingTubes = new SyncList<BreathingTubeImplant>();
```

Comment:
If you ues this you have to set dirty manly isdirty = true. When adding and removeing